package.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
add a new fields: files to include dist folder, files in this folder will be published in npm packages.
files: [
"dist"
]